src/components/controllers/page/style.module.css

```diff
@@ -7,6 +7,14 @@
 		margin-bottom: 4rem;
 	}
 
+	content-region:not([name='index']) :global(.markup) {
+		margin-top: 2.5rem;
```

Comment:
TODO: This unfortunately causes a black bar to appear beneath the header before the rest of the page switches -- `content-region` updates near immediately, while the actual content takes a bit to load. Will need to rethink.

This already exists somewhat but the change here exacerbates it.